Servers/WindowServer/Menu.h

```diff
@@ -96,6 +97,8 @@ class Menu final : public Core::Object {
 
     void draw();
     const Gfx::Font& font() const;
+    const Gfx::Font& title_font() const;
+    void set_title_font(Gfx::Font& font);
```

Comment:
Please omit the parameter name as it's implied by the type. Also, since this function is not going to mutate the `Gfx::Font`, it should be passed as a `const Gfx::Font&`.

Reply:
oh man, so silly of me :( thanks :+1: 
was trying to get better at forgetting about that. should maybe do some serenity hacking before work instead i guess..